src/fragments/lib/graphqlapi/native_common/advanced-workflows/common.mdx

```diff
@@ -175,3 +175,31 @@ import android9 from '/src/fragments/lib/graphqlapi/android/advanced-workflows/5
 import flutter8 from '/src/fragments/lib/graphqlapi/flutter/advanced-workflows/50_interceptor.mdx';
 
 <Fragments fragments={{ flutter: flutter8 }} />
+
+<InlineFilter filters={['swift']}>
+
+## Custom Request Inputs as JSON Objects
+
+If you have custom query inputs defined, you can convert the input objects to Json objects as part of the requests.
+
+```swift
+extension GraphQLRequest {
+    static func get(filter: CustomRequestFilter) -> GraphQLRequest<JSONValue> {
+        let document = """
+        ....
+        """
+        
+        let data = try? JSONEncoder().encode(filter)
+        let jsonObject = try? JSONSerialization.jsonObject(with: data!, options: [])
```

Comment:
can we avoid force casting?

Reply:
Removed force casting and add guard/throw check